contracts/CollectiveGovernance.sol

```diff
@@ -117,12 +137,44 @@ contract CollectiveGovernance is Governance, VoteStrategy, ERC165 {
         return proposalId;
     }
 
+    /// @notice Attach a transaction to the specified proposal.
+    ///         If successfull, it will be executed when voting is ended.
+    /// @dev must be called prior to configuration
+    /// @param _proposalId the id of the proposal
+    /// @param _target the target address for this transaction
+    /// @param _value the value to pass to the call
+    /// @param _signature the tranaction signature
+    /// @param _calldata the call data to pass to the call
+    /// @param _scheduleTime the expected call time, within the timelock grace,
+    ///        for the transaction
+    /// @return uint256 the transactionId
+    function attachTransaction(
```

Comment:
Is there a remove transaction function anywhere?

Reply:
No.   I can add a ticket for that and we can prioritize adding that functionality.   Its not as easy as it seems like it should be because these 'Transactions' are written to the storage.   So removing them would require us to overwrite the existing entries and shift down like deleting from the middle of an array.

Reply:
As an aside, if a transaction is dangerous or erroneously added the user still has the option to 'cancel()' the vote at this point which is pretty harmless.